core/src/main/java/com/sequenceiq/cloudbreak/service/blueprint/BlueprintService.java

```diff
@@ -409,7 +410,7 @@ public String getResourceCrnByResourceName(String resourceName) {
     @Override
     public List<String> getResourceCrnListByResourceNameList(List<String> resourceNames) {
         return resourceNames.stream()
-                .map(resourceName -> blueprintRepository.findResourceCrnByNameAndAccountId(resourceName, ThreadBasedUserCrnProvider.getAccountId()))
+                .map(resourceName -> blueprintRepository.findResourceCrnByNameAndAccountId(resourceName, ThreadBasedUserCrnProvider.getAccountId()).orElse(null))
```

Comment:
maybe it is better to throw here a not found also, isn't it?

Reply:
I was not sure to change the behaviour, since the old way only returned a null, but I guess it's safer to exit with an error.

Reply:
i guess this method is used by authz framework only so i think we can change the behavior, and in long term, we should change this method anyway to use 1 db query